nautobot/ipam/tests/test_views.py

```diff
@@ -1084,13 +1084,29 @@ def setUpTestData(cls):
         }
 
         cls.bulk_edit_data = {
-            "vlan_group": vlangroups[0].pk,
+            "vlan_group": cls.vlangroups[0].pk,
             "tenant": Tenant.objects.first().pk,
             "status": status.pk,
             "role": roles[0].pk,
             "description": "New description",
         }
 
+    def test_vlan_group_not_belong_to_vlan_locations(self):
+        """Test that a VLAN cannot be assigned to a VLAN Group that is not in the same location as the VLAN."""
+        vlan_group = self.vlangroups[0]
+        self.form_data["vlan_group"] = vlan_group.pk
+        self.form_data["locations"] = [self.locations.last().pk]
```

Comment:
since `form_data` is a class-level attribute we probably shouldn't modify it in a test as it may have unexpected impact on other test in the same class. Probably better to use a `self.form_data.copy()` in this test.